### PR TITLE
A/B test: register

### PR DESCRIPTION
**Pull request by OliverJAsh**
This is my first test! Hoping I've not missed any boilerplate. I have tested locally by enabling the switch and trying all the variants.

This does further restrict the horizontal space in the header for smaller viewports, pushing the logo down a line at an earlier breakpoint. Perhaps we should hide labels (`.control__info`) up until the `tablet` breakpoint, instead of `mobileLandscape`, to prevent the logo from being pushed down? /cc @sammorrisdesign 

As per test spec: https://docs.google.com/document/d/1Pojo-j6DlqKo-KA_Cea5P0pyd6hW7TgBHxUW1XOgeio/edit

Screenshot of the `become-a-member+free` variant:
image

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/register.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        this.audienceCriteria = 'All users who are signed out';
+        this.dataLinkNames = 'Register link';
+        this.idealOutcome = 'Conversion rate increases';
+
+        this.canRun = function () {
+            return true;
+        };
+
+        var $register = $('.js-register'),
+            $registerText = $('.control__info', $register),
```
Comment (sammorrisdesign) [inline static/src/javascripts/projects/common/modules/experiments/tests/register.js]: Maybe should be a specific js class? Like `.js-control-info`

File: static/src/stylesheets/module/nav/_brand-bar.scss [hunk 2]
```diff
@@ -98,6 +100,22 @@
     }
 }
 
+.brand-bar__item__badge {
+    // TODO: Palette?
+    background: lighten(colour(guardian-brand), 43%);
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/nav/_brand-bar.scss]: Should use `guss-colour(news-main-2, $pasteup-palette)` or `colour(news-accent)`

File: static/src/stylesheets/module/nav/_brand-bar.scss [hunk 3]
```diff
@@ -98,6 +100,22 @@
...
+    // Same as header background
+    color: colour(guardian-brand);
+    padding: 1px $gs-gutter/5;
+    margin-left: $gs-gutter/5;
+    display: none;
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/nav/_brand-bar.scss]: Doesn't this element have `u-h` on it? Shouldn't need to `display: none;` it?

**Comment by davidrapson** [comment]
Really interested to see the results of this A/B particularly how/if it affects take up for the Friend membership tier.

**Comment by sammorrisdesign** [comment]
:+1: Looks good other than comments. Might be worth getting @phamann @rich-nguyen or @sndrs to take a look at the A/B parts (although they look good to me based from my limited experience with them)

File: static/src/javascripts/projects/common/modules/experiments/tests/register.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+], function (
+    $
+) {
+    return function () {
+        this.id = 'Register';
+        this.start = '2015-03-23';
+        this.expiry = '2015-04-01';
+        this.author = 'Oliver Ash';
+        this.description = 'Testing to see if showing a register button (of some kind) will increase the conversion rate.';
+        this.audience = 0.25;
```
Comment (sammorrisdesign) [inline static/src/javascripts/projects/common/modules/experiments/tests/register.js]: The spec specifies `0.7` for this. Has it changed?

Comment (OliverJAsh) [inline static/src/javascripts/projects/common/modules/experiments/tests/register.js]: I clarified with @crifmulholland. We decided 0.7 was too large.

File: static/src/stylesheets/module/nav/_brand-bar.scss (hunk 2 above)
Comment (OliverJAsh) [inline static/src/stylesheets/module/nav/_brand-bar.scss]: Where can I go to visualise our colour palette? I'm a bit lost with it all, without some kind of visualisation.

Comment (sammorrisdesign) [inline static/src/stylesheets/module/nav/_brand-bar.scss]: [pasteup-palette demo](http://guardian.github.io/pasteup-palette/demo/)

**Pull request merged**